src/js/view/week/allday.js

```diff
@@ -52,6 +53,10 @@ function Allday(options, container) {
      */
     this.contentHeight = 0;
 
+    this.viewType = options.alldayViewType || 'scroll';
+
+    this.collapsed = (this.viewType === 'toggle');
```

Comment:
In `this.viewType is 'scroll'` case, `this.collapsed` is false. Are you sure that `this.collapsed` is false?